apps/src/code-studio/progress.js

```diff
@@ -120,11 +120,7 @@ progress.generateStageProgress = function(
   );
 
   store.dispatch(
-    mergeProgressResults(
-      _.mapValues(progressData.levels, level =>
-        level.submitted ? TestResults.SUBMITTED_RESULT : level.result
```

Comment:
Does level.submitted get used anywhere anymore? (I'm seeing it in tests and I'm not sure whether I can get rid of it)